stdlib/public/Platform/CMakeLists.txt

```diff
@@ -119,12 +121,14 @@ foreach(sdk ${SWIFT_SDKS})
       list(APPEND glibc_modulemap_target_list ${copy_glibc_modulemap_static})
     endif()
 
-    set(glibc_header_out "${module_dir}/SwiftGlibc.h")
-    handle_gyb_source_single(glibc_header_target
-        SOURCE "SwiftGlibc.h.gyb"
-        OUTPUT "${glibc_header_out}"
-        FLAGS "-DCMAKE_SDK=${sdk}")
-    list(APPEND glibc_modulemap_target_list ${glibc_header_target})
+    if(NOT ${sdk} STREQUAL FREEBSD)
```

Comment:
Can you add a comment here explaining the requirements of freebsd here (I just think it would be useful for people who don't know the BSDs).